my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'MyEnumerable'
+
+class MyList
+  include MyEnumerable
```

Comment:
Good job including `MyEnumerable` module  `MyList` class. 👍 

However, please use the correct file name to avoid the **LoadError** 🤓
![image](https://github.com/Wineshuga/enumerable/assets/78251927/84f91d5f-acee-4d72-a926-71e387e34504)

expected ✔️ 
```ruby
require_relative 'my_enumerable'
```